chore: open up contributing for external pull requests (#537)
bigcommerce/big-design: CONTRIBUTING.md

# Contributing to BigDesign

> Before contributing please refer to the [Code of Conduct](https://github.com/bigcommerce/big-design/blob/master/CODE_OF_CONDUCT.md) and we expect participants to adhere to it.

Reviewer: ```suggestion
Before contributing please read the [Code of Conduct](https://github.com/bigcommerce/big-design/blob/master/CODE_OF_CONDUCT.md) as we expect all participants to adhere to it.
```

## New Features/Changes

Before you put signifcant effort into a new feature or API changes, make sure you have [filed an issue](https://github.com/bigcommerce/big-design/issues/new?assignees=&labels=&template=feature---improvement-request.md&title=). This allows us to have a conversation around the subject and come to an agreement around the proposed solution.

If your change just fixes a bug, you can submit a pull request but ensure there is a [bug issue](https://github.com/bigcommerce/big-design/issues/new?assignees=&labels=&template=bug_report.md&title=) attached to your pull request.

## Your First Pull Request

A good first place to start is our list of [good first issues](https://github.com/bigcommerce/big-design/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

Once you are ready to start working on an issue, follow steps in **[Opening a Pull Request](#opening-a-pull-request)** and **[Development Workflow](https://github.com/bigcommerce/big-design/blob/master/README.md#development)**.

## Opening a Pull Request

> The core team will be monitoring for new pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation, within a timely maner.

Reviewer: ```suggestion
The core team will be monitoring for new pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation, within a timely manner.
```

* **Question**: For situations in which you need help from the BigCommerce engineering team, regarding guidance on using the BigDesign library, specific components, or other issues.

## New Features/Changes

Before you put signifcant effort into a new feature or API changes, make sure you have [filed an issue](https://github.com/bigcommerce/big-design/issues/new?assignees=&labels=&template=feature---improvement-request.md&title=). This allows us to have a conversation around the subject and come to an agreement around the proposed solution.

If you change just fixes a bug, you can submit a pull request but ensure there is a [bug issue](https://github.com/bigcommerce/big-design/issues/new?assignees=&labels=&template=bug_report.md&title=) attached to your pull request.

## Your First Pull Request

A good first place to start is our list of [good first issues](https://github.com/bigcommerce/big-design/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

Once you are ready to start working on an issue, follow steps in **[Opening a Pull Request](#opening-a-pull-request)** and **[Development Workflow](#development-workflow)**.

> ## Opening a Pull Request

Reviewer 2: @chanceaclark Do you want to link to the pull request template that you added in another PR to this? I think it would be good to do so.

Author: I added it here https://github.com/bigcommerce/big-design/pull/536

It automatically prefills the PR descriptions so leaving a link out if fine.
